homeassistant/components/nuki/lock.py

```diff
@@ -73,11 +74,6 @@ def unique_id(self) -> str:
         """Return a unique ID."""
         return self._nuki_device.nuki_id
 
-    @property
-    @abstractmethod
-    def is_locked(self):
-        """Return true if lock is locked."""
-
```

Comment:
I do not believe the abstractmethod served any purpose, as it is already present in the inherited `LockEntity`.